encrypted-media/content/drmconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "com.widevine.alpha": {
+        "drmtoday": true,
+        "serverURL": "https://lic.staging.drmtoday.com/license-proxy-widevine/cenc/",
+        "httpRequestHeaders": {
+            "dt-custom-data": "eyJ1c2VySWQiOiIxMjM0NSIsInNlc3Npb25JZCI6ImV3b2dJQ0p3Y205bWFXeGxJaUE2SUhzS0lDQWdJQ0p3ZFhKamFHRnpaU0lnT2lCN0lIMEtJQ0I5TEFvZ0lDSnZkWFJ3ZFhSUWNtOTBaV04wYVc5dUlpQTZJSHNLSUNBZ0lDSmthV2RwZEdGc0lpQTZJR1poYkhObExBb2dJQ0FnSW1GdVlXeHZaM1ZsSWlBNklHWmhiSE5sTEFvZ0lDQWdJbVZ1Wm05eVkyVWlJRG9nWm1Gc2MyVUtJQ0I5TEFvZ0lDSnpkRzl5WlV4cFkyVnVjMlVpSURvZ1ptRnNjMlVLZlFvSyIsIm1lcmNoYW50IjoiY2FibGVsYWJzIn0K"
```

Comment:
@ddorwin This is how we do authentication with my DRMToday authentication server.  This encoded data represents a rights/entitlement structure.  In a commercial deployment, this would be only be information regarding the customer/userID, contentID, and other client-specific info.  The authentication server would generate a rights token based on this information and return it to the DRMToday license server to facilitate the creation of a DRM-specific license.  For the purpose of doing testing, my authentication server is just a pass-through.  We pass the rights information in the license request.  This ensures we do not have to change the authentication server implementation if we want to modify the rights associated with a piece of content.